### PR TITLE
[IR-Bridge] Support attrs for operators: convolution, batch norm, relu

### PR DESCRIPTION
**Pull request by junrushao1994**
@tqchen @szha @sxjscience @eric-haibin-lin @leezu 

Ideally, we should stop using `Attrs`. But it may not be a good time now as the infra hasn't been done.

## Directory structure

PREFIX=mxnet/src/v3/

- Folder `$PREFIX/include/bridge`: headers for converters, like mxnet's nnvm (legacy_nnvm.h).
- Folder `$PREFIX/include/op`: operator definition, which is supposed to be independent from frontend.

### CODE REVIEW

File: src/v3/src/bridge/legacy_nnvm/attrs.cc [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    };
+    const auto &nnvm_attrs =
+        nnvm::get<mxnet::op::ActivationParam>(attrs.parsed);
+    if (op_map.count(nnvm_attrs.act_type)) {
+      return CallNode::make(op_map[nnvm_attrs.act_type], args, {});
+    }
+  }
+  LOG(INFO) << "Warning: cannot recognize NNVM operator " << op->name
+            << ", fallback to add";
+  return CallNode::make(Op::Get("add"), args, {}, {});
```
Comment (szha) [inline src/v3/src/bridge/legacy_nnvm/attrs.cc]: shall we log fatal here? 😆

File: src/v3/src/bridge/legacy_nnvm/ir.cc [hunk 2]
```diff
@@ -58,57 +65,58 @@ static void PrintIndexedGraph(const nnvm::Graph &g) {
     std::string op_name = op ? op->name : "None";
     if (input_nodes.count(i)) {
       input_cnt += 1;
-      op_name = (op ? op->name + " [input " : "[input ") + std::to_string(input_cnt) + "]";
+      op_name = (op ? op->name + " [input " : "[input ") +
+                std::to_string(input_cnt) + "]";
     } else {
       op_name = op ? op->name : "None";
     }
     std::cout << "  i = " << i << ", op = " << op_name
```
Comment (szha) [inline src/v3/src/bridge/legacy_nnvm/ir.cc]: LOG?

File: src/v3/include/ir.h [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+using tvm::relay::TypeReporter;
+
+// Relay Functors
+using tvm::relay::ExprFunctor;
+
+}  // namespace ir
+}  // namespace v3
+}  // namespace mxnet
+
+#define MNM_DEF_NODE_TYPE_INFO(TypeName, Parent) TVM_DECLARE_NODE_TYPE_INFO(TypeName, Parent)
```
Comment (szha) [inline src/v3/include/ir.h]: let's remember to revisit the name for MNM

Comment (junrushao1994) [reply src/v3/include/ir.h]: Hmmm good point, I will do another check

File: src/v3/src/bridge/legacy_nnvm/ir.cc (hunk 2 above)
Comment (junrushao1994) [reply src/v3/src/bridge/legacy_nnvm/ir.cc]: This is piece of program that basically did visualization, and will be removed once the entire pipeline is done. So probably `cout` looks more casual and unlikely to forget :-)

File: src/v3/src/bridge/legacy_nnvm/attrs.cc (hunk 1 above)
Comment (junrushao1994) [reply src/v3/src/bridge/legacy_nnvm/attrs.cc]: Then many unittests would crash because the operator is unsupported ><

Comment (szha) [reply src/v3/src/bridge/legacy_nnvm/attrs.cc]: We can disable the tests that are failing and re-enable them when we achieve better coverage.

Comment (junrushao1994) [reply src/v3/src/bridge/legacy_nnvm/attrs.cc]: Per offline discussion with @szha, I am going to write a separate Jenkins file

**Pull request merged**